pages/dkp/kommander/2.2/cli/dkp/open/dashboard/index.md

```diff
@@ -3,7 +3,7 @@ layout: layout.pug
 navigationTitle:  dkp open dashboard
 title:  dkp open dashboard
 menuWeight: 10
-excerpt: Open Kommander dashboard in your browser
+excerpt: Open DKP UI in your browser
```

Comment:
We may need to change this in the kommander code itself ...

Reply:
Thanks Matt... so should I not "resolve conversation" until we get an answer about whether this is changed in kommander code?